packages/discovery-react-components/src/components/SearchResults/components/Result/Result.tsx

```diff
@@ -119,7 +119,11 @@ export const Result: React.FunctionComponent<ResultProps> = ({
   const hasText = displayedText && !showTablesOnlyResults;
   const emptyResultContent = !(hasText || tableHtml);
 
-  const title = getDocumentTitle(result, resultTitleField);
+  let title = getDocumentTitle(result, resultTitleField);
+
+  if (Array.isArray(title)) {
```

Comment:
Since its not obvious, maybe a brief comment explaining why this is needed might be nice.